### ISSUE TITLE
Список пулов на "стартовой странице"

### ISSUE THREAD

**Issue by victorbocharov** [OpenCorpora/opencorpora#115]
На "стартовой странице" показывать только те типы пулов, которые нужнее всего размечать. + давать ссылку на страницу "Разметка", где есть все типы пулов.

**Comment by akmetainfo** [comment]
Дмитрий, насколько это issue актуально, раз закрыт https://github.com/OpenCorpora/opencorpora/issues/107 ? Кажется, этот тоже стоит закрыть.

**Comment by grandsbor** [comment]
Нет, это, кажется, про другое. На главной странице есть короткий список
случайных пулов.
